profiles/lib/destination.lua

```diff
@@ -0,0 +1,27 @@
+local Destination = {}
+
+function Destination.get_destination(way)
+  local destination = way:get_value_by_key("destination")
+  local destination_ref = way:get_value_by_key("destination:ref")
+
+  -- Assemble destination as: "A59: Düsseldorf, Köln"
+  --          destination:ref  ^    ^  destination
+
+  local rv = ""
+
+  if destination_ref and destination_ref ~= "" then
+    rv = rv .. destination_ref
+  end
+
+  if destination and destination ~= "" then
+      if rv ~= "" then
+          rv = rv .. ": "
+      end
+
+      rv = rv .. string.gsub(destination, ";", ", ")
```

Comment:
Yes, in that case you only get the `destination:ref` as in "Berlin (A5)". I'm missing a test for this combination, thanks for pointing it out! Will add one.